src/Network/Network/help/Set-AzNetworkManager.md

```diff
@@ -0,0 +1,225 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version:
+schema: 2.0.0
+---
+
+# Set-AzNetworkManager
+
+## SYNOPSIS
+Updates a network manager..
+
+## SYNTAX
+
+```
+Set-AzNetworkManager -ResourceGroupName <String> -NetworkManager <PSNetworkManager> [-AsJob]
```

Comment:
What's the operation method used to update Network Manager? If patch operation, please name this cmdlet as Update-AzNetworkManager. If put operation, we use set to name cmdlets. Please check all cmdlets.

Reply:
This is done using put. We are in the process of implementing a patch; let me check with my team. If it is ready, I will add it as a seperate 'Update' cmdlet.